pkg/meta/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.1.9
+
+* Introduce `@Unsafe` to annotate methods or values that typically require
```

Comment:
See below w/ a question about what can be annotated `@Unsafe`.

Reply:
Wouldn't that be `@unsafe` or `@Unsafe()` in practice?

Why not include a possible qualification, e.g., a constructor argument of type `String`, such that it could be indicated that one API typically requires a security audit (e.g., `@Unsafe("security-audit")`), another one could be unsafe because it runs an algorithm that uses a lot of resources (time and/or memory, e.g., `@Unsafe("memory")`), etc.

Surely we could have several different reasons for flagging any API as unsafe, so we might as well allow the mechanism to be extensible.

About the supported targets: We could say
```suggestion
* Introduce `@Unsafe` to annotate top level declarations, class member 
  declarations, or enum entries that typically require
```

This would allow all the constructs that @mehmetf mentions in the comment on line 281, and rule out type parameters, formal parameters, local functions and variables including `for` loop iteration variables, imports/exports, and `part` and `part of` directives.

You could make the case that an optional formal parameter should be allowed to carry this metadata as well: An `@unsafe` optional parameter would be one where only the default value is safe.